packaging/centos-7/build-image/inside/build-package

```diff
@@ -19,14 +19,12 @@ cd /tmp/build
 rm -rf build-dir
 mkdir build-dir
 cd build-dir
-cmake3 -DREALM_ENABLE_ENCRYPTION=1 \
-      -DREALM_NO_TESTS=1 \
-      -DCMAKE_INSTALL_PREFIX=/prod \
-      -DCMAKE_BUILD_TYPE=Release \
-      -DCMAKE_MAKE_PROGRAM=ninja-build \
-      -GNinja ..
-ninja-build
-ninja-build install/strip
+cmake3 -D CMAKE_INSTALL_PREFIX=/prod \
+       -D CMAKE_BUILD_TYPE=Release \
+       -D CMAKE_MAKE_PROGRAM=ninja-build \
```

Comment:
I'm curious why `CMAKE_MAKE_PROGRAM` is being set explicitly here. Is `-G Ninja` not sufficient for some reason?

Reply:
That's because the centos the executable is called `ninja-build` instead of `ninja` so that variable helps CMake calling it properly.